assemble/bin/accumulo

```diff
@@ -151,10 +151,23 @@ fi
 # Export the variables just in case they are not exported
 # This makes them available to java
 export JAVA_HOME HADOOP_PREFIX ZOOKEEPER_HOME LD_LIBRARY_PATH DYLD_LIBRARY_PATH
+
```

Comment:
Looking at the script changes, it seems I can run command like the following to start two instances :
  
    accumulo tserver-1    
    accumulo tserver-2

Did I read it correctly?  If so, why did you choose this style over something like `accumulo tserver --instance 1`? My guess is that its because the combo of Java and bash in this case is a nightmare :)  

I don't see this way of running multiple tserver instances documented anywhere in the script itself.  Like if just run `accumulo`, it seems like I will not see anything about this option.   I was looking around to see where that help message might go, and as AFAICT it would go in `org.apache.accumulo.start.Main`?